taipy/gui/gui.py

```diff
@@ -108,6 +109,38 @@
 from .utils.table_col_builder import _enhance_columns
 from .utils.threads import _invoke_async_callback
 
+TIMEZONE_FALLBACKS = {
```

Comment:
I would add a TODO comment to open an API where developers can extend this rather specific dataset...
This can be done in an issue though.